e2e/kubelet/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+
+	"github.com/sanity-io/litter"
+)
+
+func main() {
+	if err := run(); err != nil {
+		panic(err)
+	}
+}
+
+func run() error {
+	k8sVersion := os.Getenv("KUBE_BINARY_VERSION")
+	if k8sVersion == "" {
+		panic(fmt.Errorf("Environment variable KUBE_BINARY_VERSION is not set, check invocation script"))
```

Comment:
nit: panicking in an error-returning function

Reply:
oops, fixed